### PR TITLE
feat(chat): call ui updated

### PR DESCRIPTION
**Pull request by molimauro**
### What this PR does 📖
- Call ui updates

### Which issue(s) this PR fixes 🔨
- Resolve #

### Special notes for reviewers 🗒️
Added the first username letter in the Media user box, lmk if it doesnt look good. We should put the Satellite icon later

### CODE REVIEW

File: src/components/media/styles.scss [hunk 1]
```diff
@@ -23,23 +22,26 @@
...
-    top: 0.5rem;
-    right: 0.5rem;
-  }
-  .settings-toggle {
-    position: absolute;
-    top: 0.5rem;
-    left: 0.5rem;
+    #media-content {
+      flex: 1;
+      margin-top: 12px;
```
Comment (InfamousVague) [inline src/components/media/styles.scss]: we should use consistent padding and margin so either 0.5rem or 1rem, the gap let's set to 1rem

File: src/ui_kit/src/profile_picture/styles.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .pfp {
   background: var(--theme-text-muted);
   background-size: cover !important;
-  border-radius: 20px;
+  border-radius: 50%;
```
Comment (InfamousVague) [inline src/ui_kit/src/profile_picture/styles.scss:4]: Either of these work, doesn't matter to me 😁

File: src/components/media/media/styles.scss [hunk 3]
```diff
@@ -1,19 +1,54 @@
...
+    .pfp-container {
+      .pfp {
+        height: 80px;
+        width: 80px;
       }
     }
+
+    .placeholder {
+      position: absolute;
+      font-size: 2rem;
```
Comment (InfamousVague) [inline src/components/media/media/styles.scss]: we should use one of the font sizes from the theme so `var(--text-size)` may be best, there is also smaller

File: src/components/media/media/styles.scss [hunk 4]
```diff
@@ -1,19 +1,54 @@
...
+    position: absolute;
+    inset: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+
+    .pfp-container {
+      .pfp {
+        height: 80px;
```
Comment (InfamousVague) [inline src/components/media/media/styles.scss]: let's do 40x40px for now just to keep it the same as everything else, unless it's larger on profile then we can use that

File: src/components/main/compose/messages/styles.scss [hunk 5]
```diff
@@ -28,7 +28,7 @@
     width: fit-content;
     align-self: center;
     padding: 0.5rem;
-    margin: 1rem 0;
+    margin: 16px 0;
```
Comment (InfamousVague) [inline src/components/main/compose/messages/styles.scss]: for now let's omit this just so our margins are consistent, makes it easier to find and replace later if/when we change it

**Review by InfamousVague — CHANGES_REQUESTED** [review]
small changes then looks good for QA

**Comment by molimauro** [comment]
Done, thanks for the feedback!!

**Comment by phillsatellite** [comment]
Noticed while testing that we only show 1 user in the call and a side note from Matt, the call should be able to display many users in a grid shrinking the users but maintaining aspect ratio.

image

**Comment by molimauro** [comment]
I can increase the number of users but they will keep the same size regardless of the resize in place. I think we should merge this one and focus later on the user media resize, what do you think?

**Comment by stavares843** [comment]
also agree

**Comment by phillsatellite** [comment]
Sounds good to me!

**Review by phillsatellite — APPROVED** [review]

**Review by InfamousVague — APPROVED** [review]

**Pull request merged**